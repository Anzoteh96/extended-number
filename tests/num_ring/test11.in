-17912063302832048106026700961456
190429337853494781145224025022648
13937465123253207681687820345045
