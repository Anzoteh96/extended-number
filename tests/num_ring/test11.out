Enter the number base b:
Enter two numbers n and m:
b is 17912063302832048106026700961456
n mod b is 11308704825174300084957015408088
m mod b is 13937465123253207681687820345045
n + m is 7334106645595459660618134791677
n - m is 15283303004753140509295896024499
n * m is 8532174349439277680366697212184
n ** m is 1418999501202556522293141092480
